src/assets/server/rest/v1/docs/e-mobility-oas.json

```diff
@@ -3634,6 +3761,42 @@
           ]
         }
       },
+      "StartDateTime": {
+        "in": "query",
+        "name": "StartDateTime",
+        "description": "Start Date (ISO format)",
+        "required": false,
+        "schema": {
+          "type": "string"
```

Comment:
date or date-time type is not a supported type?

Reply:
It's passed in the query string so as a string :-)